app/views/verify/finance_other/new.html.slim

```diff
@@ -28,12 +28,10 @@ p.mt-tiny.mb0 = t('idv.messages.finance.intro_account')
                  t('forms.buttons.continue'),
                  disabled: true,
                  class: 'col-12 mb3 js-finance-submit'
-
-.mb-tiny.bold = t('idv.messages.finance.no_account')
+.mb1
+  = link_to t('idv.messages.finance.no_account'), verify_finance_path, class: 'tiny'
 p.mb3
   = t('idv.messages.finance.no_account_info')
-  '
-  = link_to t('idv.form.use_ccn'), verify_finance_path
 
-= render 'shared/cancel', link: verify_cancel_path
+= render 'shared/cancel', link: verify_cancel_path, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.